### PR TITLE
Delete take part’s frontend parts

### PR DESCRIPTION
**Pull request by fofr**
Deletes:
* styles
* template
* controller
* test assertions for rendered `show` pages

While the controller isn’t needed as Whitehall is no longer serving these routes, the `route` itself needs to be kept. We rely on the routes defined in config/routes.rb to generate methods which give us the public URLs for those things. If we want to remove the routes, which we should, then we need to create substitutes for those methods ourselves.

A separate story to address this: https://trello.com/c/wvdvAKek/

For now the route has been left in, with a comment. @jamiecobbett and I considered the work to resolve this too large, and shouldn't block this clean-up PR. It will affect any format being migrated where we wish to remove old frontend code in Whitehall.

https://trello.com/c/TLCjyX9x/269-take-part-remove-any-frontend-code-in-whitehall

cc @dsingleton @boffbowsh @gpeng @benlovell

### CODE REVIEW

File: features/take-part-pages.feature [hunk 1]
```diff
@@ -15,16 +15,15 @@ Feature: Administering take part pages
     image: image
 
   They can be reordered so the /get-involved index lists them in a pleasing manner
-  
+
   Background:
     Given I am a GDS editor
-  
+
```
Comment (benlovell) [inline features/take-part-pages.feature]: :sparkles:

**Comment by benilovj** [comment]
As mentioned in person, it may be good to move `app/views/take_part_pages/_take_part_page.html.erb` to `app/views/home/` so it's closer to `get_involved.html.erb`, which is the only place where it's used.

**Comment by fofr** [comment]
@benilovj Done in 542a2c7

**Comment by benilovj** [comment]
:+1:

**Pull request merged**